### DISCUSSION TITLE
Deal not requiring monitoring and not buying first Safety Order: trailingstoploss_tp.ini

### DISCUSSION THREAD

**Discussion by ifmvaiea** [cyberjunky/3commas-cyber-bots#202] [General]
Hello,

My first Safety Order is not being placed and the deal is not being monitored while the current profit is below my assumed `activation-percentage` and the re-calculated Add Funds percentage. I have one Paper Trade deal open and the script reports:

**DEBUG - Bot: USDT_DATA/2216668813 has filled all 0 Safety Orders.
DEBUG - Deleting old deals from bot 12490053 except 2216668813
DEBUG - Bot (12490053) has 1 deal(s) of which 0 require monitoring.**

Here are my settings:

`safety-config = [{"activation-percentage": "2.1","activation-so-count": "0","initial-buy-percentage": "0.2","buy-increment-factor": "0.6"}]`

I'm assuming that when real profit hits -2.33% a Safety Order would be placed at -2.32%, but I'm starting to suspect my settings are wrong. Today the price dropped well below my assumed target and there were no Safety Orders placed. I just keep getting the same message that the deal doesn't require monitoring. Is there a way to tell what the Add Funds percentage is at any given time? 

Thank you!

**Comment by amargedon** [comment]
The first line of the debug log tells that the max of zero safety orders has been reached. I suspect the deal (and maybe bot) in 3C has been configured with the max number of safety orders to zero?
